src/pick.c

```diff
@@ -729,3 +792,15 @@ free_choices(void)
 	free(choices.v);
 	free(input.string);
 }
+
+int
+isu8cont(unsigned char c)
```

Comment:
On Fri, Apr 08, 2016 at 01:41:19AM -0700, Calle Erlandsson wrote:
> > @@ -729,3 +792,15 @@ free_choices(void)
> >  	free(choices.v);
> >  	free(input.string);
> >  }
> > +
> > +int
> > +isu8cont(unsigned char c)
> 
> Does this function return whether or not the byte is the continuation of an utf-8 character?

It returns 1 if the given byte is a UTF-8 continuation byte.